src/Kernel32.Shared/NTStatusException.cs

```diff
@@ -78,8 +80,9 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
         /// Gets the message associated with the given <see cref="NTSTATUS"/>.
         /// </summary>
         /// <param name="status">The <see cref="NTSTATUS"/> for the error.</param>
+        /// <param name="dwLanguageId">The language identifier for the message</param>
         /// <returns>The description of the error.</returns>
-        private static string GetMessage(NTSTATUS status)
+        private static string GetMessage(NTSTATUS status, int dwLanguageId = 0)
```

Comment:
I think we'd want to avoid the `= 0` default value on the private members in order to avoid mistakes where we don't propagate the value from other members of the class.